MmSupervisorPkg/Core/Relocate/Relocate.c

```diff
@@ -698,7 +698,11 @@ SetupSmiEntryExit (
   //
   // Initialize address fixup
   //
-  PiSmmCpuSmiEntryFixupAddress ();
+
+  // This is only necessary when using the MmSupervisorCore SmiEntry.nasm.  Ignore it with SEA.

```

Comment:
nit: you probably do not want to mention SEA in the comment. Just that if feature lib has its own entry code, no fixup is needed.

Reply:
Okay.  I fixed the comment to be implementation agnostic.